src/chainparams.cpp

```diff
@@ -261,10 +261,10 @@ class CTestNetParams : public CChainParams {
         };
 
         chainTxData = ChainTxData{
-            // Data from RPC: getchaintxstats 4096 0000000000004ae2f3896ca8ecd41c460a35bf6184e145d91558cece1c688a76
-            /* nTime    */ 1625727096,
-            /* nTxCount */ 60408943,
-            /* dTxRate  */ 0.08379062270367649,
+            // Data from RPC: getchaintxstats 4096 00000000d18cfe81cbeea665076807789bd8f831d557632e635bc6e3c003069e
+            /* nTime    */ 1645635119,
+            /* nTxCount */ 44718,
```

Comment:
This is `window_tx_count`, `txcount` is `62226341`

Reply:
Good catch! Updated and re-pushed.